frontend/src/components/forms/AccountEditor.tsx

```diff
@@ -28,33 +30,38 @@ export default function AccountEditor({ userInfo }: { userInfo: UserInfo }) {
       return;
     }
 
-    const res = await fetch("https://jsonplaceholder.typicode.com/users/1", {
-      method: "PATCH",
-      body: JSON.stringify({
-        email: userInfo.email,
-        password: newPassword,
-      }),
-    }).catch((err) => {
-      console.log(err);
-      return {
-        status: 500,
-        ok: false,
-      };
-    });
-
-    if (!res.ok) {
-      setAccountMessage("An error occured, please try again later");
-    } else {
-      setAccountMessage("Password Updated!");
-      setNewPassword("");
+    try {
+      await userService.updateUserPassword(
+        uid,
+        oldPassword,
+        newPassword,
+      );
       setUpdateCount(updateCount + 1);
+      setAccountMessage("Update password successfully");
+    } catch (error) {
+      if (error instanceof Error) {
+        const errorMessage = error.message;
+        setAccountMessage(errorMessage);
+      } else {
+        setAccountMessage("Unknown error updating password, please try again");
+      }
+    } finally {
+      setNewPassword("");
+      setOldPassword("");
     }
   };
 
   return (
     <form className="flex w-1/2 flex-col gap-4" key={updateCount}>
       <Input type="email" isDisabled label="Email" value={userInfo.email} />
       <PasswordInput
+        label={"oldPassword"}
```

Comment:
The label here is inconsistent with "Confirmed password" field, see the example screenshot:
![Uploading image.png…]()

Reply:
please reupload the image i cant see it 
![image](https://github.com/tryyang2001/CS3213-Frontend-Management-System/assets/64810838/fc314a65-8b58-42e6-8497-053ae0b51e27)

Reply:
![image](https://github.com/tryyang2001/CS3213-Frontend-Management-System/assets/69421979/1654f2c4-7692-4682-94ff-26744a850dd8)

Reply:
resolved